### PR TITLE
add options on unittest script

### PR DESCRIPTION
**Pull request by ticoann**
will have option to deploy certain version, from certain repository and branch.
Also added code for only updating the source code when previously deployed unittest exist
https://github.com/dmwm/WMCore/wiki/Setup-wmcore-unittest

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, @ericvaandering, Please review

File: test/deploy/deploy_unittest.sh [hunk 1]
```diff
@@ -1,13 +1,69 @@
...
\ No newline at end of file
+
+VERSION=1.0.7.pre3
+REPOSITORY=dmwm
+BRANCH=
+UPDATE=false
+
+deploy_agent() {
+
+    git clone git@github.com:dmwm/deployment.git
```
Comment (amaltaro) [inline test/deploy/deploy_unittest.sh]: Can you change it to https (e.g.: https://github.com/USER/REPO.git). One password less :)
The same for the git clone below

**Comment by amaltaro** [comment]
Then 2 more comments here not to pollute the other "view" :-)
 1. in the case statement, you should shift twice (-v VERSION)
 2. maybe add an example line on the top and that's all

**Comment by ticoann** [comment]
Thanks Alan, I updated as you suggested, about the shift there is one one shift outside the case so it should work fine.

**Comment by ticoann** [comment]
Merging it now for the Alex and Tapas to test. Alan, please review anyway I will fix if you find anything more.

**Pull request merged**